diff --git a/lib/generate-code.js b/lib/generate-code.js
index 5ef91438373ac07d2811887a2e88b0b3cb48884c..a3f9d13cd1b1f8c93a884313889f3a3bf90f6e81 100644
--- a/lib/generate-code.js
+++ b/lib/generate-code.js
@@ -54,7 +54,7 @@ module.exports = function generateCode(
     )
 
     node.add(
-      `__options__.render = render\n` +
+      `\n__options__.render = render\n` +
         `${namespace}.staticRenderFns = staticRenderFns\n`
     )
 