diff --git a/dist/provider.js b/dist/provider.js
index e1eb11d..a815f1c 100644
--- a/dist/provider.js
+++ b/dist/provider.js
@@ -1800,14 +1800,8 @@ const CovBranch = requireBranch();
 const CovFunction = require_function();
 const CovSource = requireSource();
 const { sliceRange } = requireRange();
-const compatError = Error(`requires Node.js ${require$$9.engines.node}`);
-const { readFileSync } = require$$10;
-let readFile = () => { throw compatError };
-try {
-  readFile = require('fs').promises.readFile;
-} catch (_err) {
-  // most likely we're on an older version of Node.js.
-}
+const { readFileSync } = require$$9;
+const readFile = promises.readFile;
 const { TraceMap } = requireTraceMapping_umd();
 const isOlderNode10 = /^v10\.(([0-9]\.)|(1[0-5]\.))/u.test(process.version);
 const isNode8 = /^v8\./.test(process.version);
