diff --git a/dist/vue-inline-svg.js b/dist/vue-inline-svg.js
index d41be2f1da8615dac2242313a629c6b8b93ac263..04cae73f0fd7c20a02faf8f68ec3f360287c0cb1 100644
--- a/dist/vue-inline-svg.js
+++ b/dist/vue-inline-svg.js
@@ -1,5 +1,5 @@
 (function (global, factory) {
-    typeof exports === 'object' && typeof module !== 'undefined' ? factory(exports) :
+        typeof exports === 'object' && typeof module !== 'undefined' ? function() { Object.defineProperty(exports, "__esModule", { value: true }); module.exports.default = factory(require('vue')) }() :
     typeof define === 'function' && define.amd ? define(['exports'], factory) :
     (global = typeof globalThis !== 'undefined' ? globalThis : global || self, factory(global.VueInlineSvg = {}));
 })(this, (function (exports) { 'use strict';