diff --git a/dist/vue-inline-svg.js b/dist/vue-inline-svg.js
index 293dfd0e89d2de6b38cde450c8ab910ce2dd344f..050c788fb9b304f82d7039fcf3ad022992837578 100644
--- a/dist/vue-inline-svg.js
+++ b/dist/vue-inline-svg.js
@@ -1,5 +1,5 @@
 (function (global, factory) {
-    typeof exports === 'object' && typeof module !== 'undefined' ? module.exports = factory(require('vue')) :
+    typeof exports === 'object' && typeof module !== 'undefined' ? function() { Object.defineProperty(exports, "__esModule", { value: true }); module.exports.default = factory(require('vue')) }() :
     typeof define === 'function' && define.amd ? define(['vue'], factory) :
     (global = typeof globalThis !== 'undefined' ? globalThis : global || self, global.VueInlineSvg = factory(global.Vue));
 })(this, (function (vue) { 'use strict';