diff --git a/dist/vue-gettext.js b/dist/vue-gettext.js
index fe8eff3af09e652fda57c3a325a19967442f6ff0..c23cc96fd168083ff9b8dc31c8f84830dd7bba0f 100755
--- a/dist/vue-gettext.js
+++ b/dist/vue-gettext.js
@@ -888,8 +888,8 @@

     shareVueInstance(Vue);

-    Override(Vue, languageVm);
-
+    Vue.prototype.$language = languageVm;
+
     Config(Vue, languageVm, options.silent, options.autoAddKeyAttributes, options.muteLanguages);

     translate.initTranslations(options.translations, Vue.config);
